bundle/docsgen/markdown.go

```diff
@@ -93,7 +94,21 @@ func formatDescription(a attributeNode) string {
 		} else if s != "" {
 			s += ". "
 		}
-		s += fmt.Sprintf("See [_](#%s).", a.Link)
+		s += fmt.Sprintf("See [_](#%s).", cleanAnchor(a.Link))
 	}
 	return s
 }
+
+func cleanAnchor(s string) string {
+	s = strings.ReplaceAll(s, "<", "")
+	s = strings.ReplaceAll(s, ">", "")
+	s = strings.ReplaceAll(s, ".", "")
+
+	return s
+}
+
+func escapeBrackets(s string) string {
+	s = strings.ReplaceAll(s, "<", "\\<")
+	s = strings.ReplaceAll(s, ">", "\\>")
+	return s
+}
```

Comment:
Can you include a few tests with cases that you need to address as examples?

The replacements look quite broad. Are these characters categorically rejected?